File: Aula_02/Java/src/Fibonacci_facil.java
import java.util.Scanner;

public class Fibonacci_facil {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        int numero = sc.nextInt();

        if (numero == 1){
            int primeiro_valor = 0;
            System.out.println(primeiro_valor);
        } else {
            int primeiro_valor = 0, segundo_valor = 1;
            System.out.print(primeiro_valor + " " + segundo_valor + " ");
            for (int i = 0; i < numero-2; i++){
                int valor = primeiro_valor + segundo_valor;
                System.out.print(valor + " ");
                primeiro_valor = segundo_valor;
                segundo_valor = valor;
            }
        }

        sc.close();
    }
}

File: Aula_02/C++/Fibonacci_facil.cpp
#include <iostream>

int main(){
    int numero;
    std::cin >> numero;
    if (numero == 1){
        int primeiro_valor = 0;
        std::cout << primeiro_valor << std::endl;
    } else {
        int primeiro_valor = 0, segundo_valor = 1;
        std::cout << primeiro_valor << " " << segundo_valor << " ";
        for (int i = 0; i < numero-2; i++){
            int valor = primeiro_valor + segundo_valor;
            std::cout << valor << " ";
            primeiro_valor = segundo_valor;
            segundo_valor = valor;
        }
        std::cout << std::endl;
    }

    return 0;
}